### DISCUSSION TITLE
allow installing non bom stock item

### DISCUSSION THREAD

**Discussion by aep** [inventree/InvenTree#6651] [General]
Hi,

i'm using inventree to track servers and components in it.
it's a bit inconvenient to edit the BOM to include all possible components, like all 14 different drive types.

is there a way to disable the "must be on BOM" restriction?

**Comment by matmair** [comment]
InvenTree is a PLM, not a Asses manage solution so no and not probable to be added on stock instances. The definition is am using is [here](https://docs.inventree.org/en/stable/concepts/terminology/), we recommend other solutions on that page too.

**Comment by SchrodingersGat** [comment]
@aep I would agree in principle with @matmair - this is a bit outside the scope / intent of this feature.

However it would not "hurt" to add this behind a (default off) setting. But this is not a development priority. If you want to sponsor this I'd be happy to look into it.
